lib/twine/cli.rb

```diff
@@ -89,6 +89,12 @@ def self.parse(args)
           end
           options[:output_encoding] = e
         end
+        opts.on('-b', '--deliberate', 'Validate the strings file before formatting it') do
```

Comment:
Why `--deliberate`? Is this a common flag name for this type of check? I was thinking `--validate`